### ISSUE TITLE
Modal close icon is not visible

### ISSUE THREAD

**Issue by NikLP** [localgovdrupal/localgov_base#714]
Per title - an example is, set a webform to open in a modal, the X icon is visible in the source code but is not apparent in the front-end. 

The same icon is visible if switching to Gin theme (for example). 
Discovered via a subtheme, but on inspection same scenario apparent in localgov_base.

Modal in Gin
![Image](https://github.com/user-attachments/assets/085d6e00-b3f6-4231-b7db-dd0c24f248bf)

Modal in Base theme
![Image](https://github.com/user-attachments/assets/03962d0f-1ee4-4111-bfbb-bfe5f9cd7167)

Source code sample indicating presence of Close elements
![Image](https://github.com/user-attachments/assets/8ce73808-adb7-43d1-ad2e-e4f9edde0d22)

**Comment by markconroy** [comment]
On a default installation of LGD with demo_content module enabled, I am able to see the close button.

I have localgov_base set as my admin theme in this screenshot.

<img width="1068" alt="Image" src="https://github.com/user-attachments/assets/66311219-91ca-4260-affe-18c2018e4e6d" />

@NikLP Can you provide any more details about how you got to the error you are reporting? What's your exact config for this or steps to reproduce?

**Comment by NikLP** [comment]
Sorry @markconroy - apparently can no longer reproduce, must have been me or 'fixed' in a recent version. Will reopen if this resurfaces.
